controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
I see. This seems like a lot of work to get defaults working (3 layers: `def.json` to policy to `body server control`). Would the following work better? It only uses two layers (`def.json` to `body server control`).

```
body server control
{
  maxconnections => ifelse( isvariable("def.server[maxconnections]"), "$(def.server[maxconnections])", "200");
}
```

I think that serves the use case you mentioned and the case where you want to use classes, but I'm maybe missing others?